core/api/src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -219,7 +215,7 @@ interface ILightningService {
   }): Promise<RawRoute | LightningServiceError>
 
   registerInvoice(
-    args: RegisterInvoiceArgs,
+    args: RegisterInvoiceArgs & { lndsAndPubkeys: LndAndPubkey[] },
```

Comment:
this is not a good interface update, this method is an abstraction that must not require to pass lnds info or "available nodes" (this must be hidden inside service). Probably is better to use pubkey from RegisteredInvoice

Reply:
Ah true, missed this! I got distracted by trying to find a way to test this from outside the service. Fixed now